models/evc.py

```diff
@@ -949,6 +952,11 @@ def _prepare_flow_mod(self, in_interface, out_interface, queue_id=None):
         }
         if self.sb_priority:
             flow_mod["priority"] = self.sb_priority
+        else:
```

Comment:
This approach also works, but then if `sb_priority` is set and we have a symmetrical tag/no-tag EVC then they'd all have the same priority. Should we try to implement your suggestion to introduce another optional `sb_priority` at the `uni_<x>` level and then update the API spec accordingly? If we introduce that, then I think this method here we should probably parametrize the priority, that way, whoever is calling it (the other `_prepare.*` methods) they set the priority accordingly, otherwise it would be hard to differentiate all the cases in this `_prepare_flow_mod` common method here.

We could also proceed with this current proposed implementation and then evolve it later, it's up to you @Alopalao and @italovalcy  to decide and align about it considering the use cases we'll go for. Once the decision is made, let's also make sure to ship e2e tests.

Reply:
I thought `sb_priority` should overwrite any default priority of EVPL or EPL. I thought about adding the `sb_priority` to the UNIs but that would have raised more questions:
- What happens when all three `sb_priority` are sent in the request?
- The `sb_priority` outside the UNIs should be used for NNIs?
- If not, should I remove it or try to not accept a request with the three `sb_priority` in it?

Reply:
I think so @Alopalao , `sb_priority` outside of the the UNIs should be used for NNIs, and then `sb_priority` inside of the UNIs, if defined use whatever that value is, otherwise use the default value that you introduced. I'll recommend that you sync that with @italovalcy first about it, after all having this new flexibility is great, but if we won't potentially use in production for now then we can proceed with what's being delivered, and only in the future support more customization to the the priority. Thanks.

Reply:
great discussion @Alopalao and @viniarck . It sounds like a lightweight blueprint would be very beneficial to help clarify all those questions (besides providing good documentation for the future). I believe for now it is okay to overwrite the default priority of EVPL or EPL when the request comes with a `sb_priority` (i.e., if the network operator / requestor explicitly asked for a certain sb_priority, let's take that request as the baseline instead of our default values)

Reply:
Sounds good, I've captured it here https://github.com/kytos-ng/kytos/issues/304 just so we have it on our backlog.